### ISSUE TITLE
[a11y] AppBar titles should be marked as Header

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#16393]

**Comment by jonahwilliams** [comment]
@goderbauer Does this apply to cupertino NavBars as well?

**Comment by goderbauer** [comment]
I'd look at a native cupertino NavBar with the accessibility inspector and see if the flag is set there.
